crates/plugin/src/config.rs

```diff
@@ -48,8 +48,12 @@ pub struct Metrics {
 #[derive(Debug, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct Accounts {
+    #[serde(default)]
     pub owners: HashSet<String>,
 
+    #[serde(default)]
```

Comment:
```suggestion
    pub owners: HashSet<String>,

```
I actually explicitly removed the `#[serde(default)]` attribute on most config keys because we ran into multiple issues previously with that attribute making typos in the config harder to detect.